pkg/cmd/server/origin/controller/network_unsupported.go

```diff
@@ -13,5 +13,6 @@ type SDNControllerConfig struct {
 }
 
 func (c *SDNControllerConfig) RunController(ctx ControllerContext) (bool, error) {
-	return false, fmt.Errorf("SDN not supported on this platform")
+	glog.Errorf("SDN not supported on this platform")
```

Comment:
Oops, I was thinking this only even got called if the config specified `networkPluginName: redhat/openshift-ovs-multitenant`, etc. But that's checked elsewhere.

I think this should do:

    if sdn.IsOpenShiftNetworkPlugin(c.NetworkConfig.NetworkPluginName) {
            return false, fmt.Errorf("SDN not supported on this platform")
    } else {
            return false, nil
    }

(where `sdn` is `"github.com/openshift/origin/pkg/sdn"`)

Then if master-config.yaml says to use the SDN plugin, it will fail to start. But otherwise it will just log `Skipping "openshift.io/sdn"`

Reply:
the linux controller doesn't check `sdn.IsOpenShiftNetworkPlugin`... should it?